addons/mail/static/src/models/thread/thread.js

```diff
@@ -1880,6 +1882,12 @@ registerModel({
             required: true,
         }),
         channel_type: attr(),
+        /**
+         * States whether current user can add other followers to this thread.
+         */
+        canAddFollowers: attr({
```

Comment:
I would suggest to name it `hasWriteAccess` which is actually what it contains, instead of `canAddFollowers` which is just a consequence.
(ideally I would like to also have the idea that it's "for current user" in the field name but I don't have a good suggestion for that part that wouldn't be too long)

Also make sure to sort it alphabetically with other fields.